src/main/scala/beam/router/r5/R5RoutingWorker.scala

```diff
@@ -696,12 +698,13 @@ class R5RoutingWorker(workerParams: WorkerParameters) extends Actor with ActorLo
               for ((beamLeg, index) <- tripWithFares.trip.legs.zipWithIndex) yield {
                 var cost = tripWithFares.legFares.getOrElse(index, 0.0)
                 val age = routingRequest.attributesOfIndividual.flatMap(_.age)
-                val ids = beamServices.agencyAndRouteByVehicleIds.get(
-                  beamLeg.travelPath.transitStops.fold(vehicle.id)(_.vehicleId)
-                )
-                cost =
-                  ids.fold(cost)(id => beamServices.ptFares.getPtFare(Some(id._1), Some(id._2), age).getOrElse(cost))
-
+                val vehicleId = beamLeg.travelPath.transitStops.fold(vehicle.id)(_.vehicleId)
```

Comment:
Just as verification, is the logic here that if it's the same vehicle then the cost will be the same - so save the cycles?